gradle.properties

```diff
@@ -6,7 +6,7 @@ kotlin.native.ignoreDisabledTargets=true
 
 
 # FIXME - see https://youtrack.jetbrains.com/issue/KTIJ-21583/HMPP-1-6-20-breaks-autocomplete-in-multiplatform-composite-build
-kotlin.mpp.hierarchicalStructureSupport=false
+#kotlin.mpp.hierarchicalStructureSupport=false
```

Comment:
hmm this wasn't intentional, I was trying to get things building again and they do. If everything works again though for everyone else we can probably remove this completely. Which we should do anyway as this project _should_ be compiled as a HMP project and this setting should have only been used locally (e.g. via `local.properties`) in retrospect.